apps/src/javalab/Backpack.jsx

```diff
@@ -217,6 +217,12 @@ class Backpack extends Component {
     }
   };
 
+  handleKeyDown = event => {
```

Comment:
could we turn this into a helper component that is just a div that is closed on 'escape' and you can put any children inside it?

Reply:
Hmm yeah, I was thinking of creating a HOC sort of like `onClickOutside` at first but that seemed like a little much for this use case. However, just creating a wrapper component seems to be straightforward, I can try that

Reply:
Reworked this to use a wrapper component - let me know if that looks better!